build-system/pr-check.js

```diff
@@ -385,13 +385,14 @@ const command = {
     }
     // Unit tests with Travis' default chromium in coverage mode.
     timedExecOrDie(cmd + ' --headless --coverage');
-    if (process.env.TRAVIS) {
-      // A subset of unit tests on other browsers via sauce labs
-      cmd = cmd + ' --saucelabs_lite';
-      startSauceConnect();
-      timedExecOrDie(cmd);
-      stopSauceConnect();
-    }
+    // TODO(erwinm): temporary until saucelabs is back up
+    //if (process.env.TRAVIS) {
+      //// A subset of unit tests on other browsers via sauce labs
+      //cmd = cmd + ' --saucelabs_lite';
+      //startSauceConnect();
```

Comment:
Lint might complain that this function isn't being used. If so, comment out the function too.